pkgs/development/libraries/libvirt/default.nix

```diff
@@ -60,15 +60,12 @@ in stdenv.mkDerivation rec {
       --replace 'lxc_path,' '"/run/libvirt/nix-emulators/libvirt_lxc",'
 
     patchShebangs . # fixes /usr/bin/python references
-    substituteInPlace src/util/virjsoncompat.c --replace \
-        '"libjansson.so.4"' '"${jansson}/lib/libjansson${stdenv.targetPlatform.extensions.sharedLibrary}"'
-   '';
+  '';
 
   configureFlags = [
     "--localstatedir=/var"
     "--sysconfdir=/var/lib"
     "--with-libpcap"
-    "--with-qemu"
```

Comment:
I'd keep ```--with-qemu```, 
although it is default in ```4.5.0``` but conditional in ```4.6.0``` and possible ```4.7.0``` and it is better to be sure we build ```libvirt``` with ```qemu``` than to let ```pkg-config``` decide

Reply:
I mean, on ```4.5.0 -> 4.7.0``` we likely will hit again this issue https://github.com/NixOS/nixpkgs/pull/44566#issuecomment-412222753

Reply:
Thanks, we may as well do that now :+1: